src/jdk.compiler/share/classes/com/sun/source/doctree/DocTreeVisitor.java

```diff
@@ -287,6 +287,22 @@ default R visitProvides(ProvidesTree node, P p) {
      */
     R visitSince(SinceTree node, P p);
 
+    /**
+     * Visits a {@code SpecTree} node.
+     *
+     * @implSpec Visits the provided {@code SpecTree} node
+     * by calling {@code visitOther(node, p)}.
+     *
+     * @param node the node being visited
+     * @param p a parameter value
+     * @return a result value
+     *
+     * @since 19
+     */
+    default R visitSpec(SpecTree node, P p) {
```

Comment:
Nit: this method breaks the alphabetic order of visitor methods. Although `visitOther` also breaks it, we should not address it in this PR.

Reply:
Actually, fixed. Another merge issue.